### ISSUE TITLE
module does not maintain type of objects through sets/gets - specifically booleans

### ISSUE THREAD

**Issue by nagey** [grevory/angular-local-storage#203]
When storing a variable whose value is ```true``` or ```false``` in the module, when you go to retrieve that selfsame item later, you will get back ```"true"``` or ```"false"``` 

This is an unfortunate behavior. Easy enough to work around, but unfortunate nevertheless.

Can easily be solved by popping everything stored into an object with the actual value stored with a key named ```value``` like so (super-oversimplified version):
```
function set(key, value) {
   var thingAboutToBeStored = { "value": value };
   localStorage[key] = thingAboutToBeStored;
}

function get(key) {
  var value = JSON.parse(localStorage[key]);
  return value.value;
}
```

**Comment by matthew-mahony** [comment]
I found this to be a pain because you can't use bind with bool variables due to this issue.

**Comment by thmarti** [comment]
+1

**Comment by idooo** [comment]
This is the standard behaviour of JSON.parse() which used by angular.fromJson function used in this get(). However there is reviver param fn in JSON.parse() that can be used to change the way transformation works.

I've added PR but not so sure if it would be merged or not. Unfortunately these changes are breaking.

**Comment by a8m** [comment]
Closed via #220
